redhat-cop/infra-ansible
# Added ability to install Nagios from source.

### What does this PR do?
Brief explanation of the code or documentation change you've made.
Role now includes the following.
 - A proper README file with variables explained, along with how the playbook and inventory should be configured to complete a successful deployment.

### How should this be tested?
Automated tests are preferred, but not always doable - especially for infrastructure. Include commands to run your new feature, and also post-run commands to validate that it worked. (please use code blocks to format code samples)

Build a new RHEL7 system and register it to CDN
Create playbook and inventory according to README examples.
Run playbook against new VM built.
Ensure hostname of VM is resolvable then open browser to URL and use credentials set or default values.

### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
resolves #<number>

None open, just a role that needed some refinement and enhancement.

### Other Relevant info, PRs, etc.
Please provide link to other PRs that may be related (blocking, resolves, etc. etc.)

### People to notify
cc: @redhat-cop/infra-ansible
@redhat-cop/infra-ansible
